common/culture/traditions/00_ritual_traditions.txt

```diff
@@ -350,12 +348,19 @@ tradition_sacred_groves = {
 	}
 	
 	is_shown = {
-		scope:character = { has_government = tribal_government }
+		scope:character = { 
+			OR = {
+				has_government = tribal_government 
+				religion = { is_in_family = rf_life }
+				religion = { is_in_family = rf_elemental }
```

Comment:
You meant orcs by this?

Reply:
Ideally only the ape, loa, pandaren, water, shamanism groups. Right now rf_elemental also includes the air, earth, fire and lizard groups. I can include a NOR state for those specific faiths but their children subcultures would still have access. Is there a way to block specific groups instead?

Reply:
I'm not sure the tradition fits since it talks about trees and forests.

Reply:
I guess the only group of the above that has a strong basis for revering the woods as a core part of their philosophy would be the pandaren religion (spirit element) due to its emphasis on balance. I'll exclude the rest then?